### PR TITLE
[Soundcloud] Added -original quality thumbnail (closes #20554)

### LINKED ISSUE #20554: youtube-dl not downloading high quality thumbnails with SoundCloud
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`)
- Use the *Preview* tab to see what your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2019.04.01*. If it's not, read [this FAQ entry](https://github.com/ytdl-org/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2019.04.01**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through the [README](https://github.com/ytdl-org/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/ytdl-org/youtube-dl#faq) and [BUGS](https://github.com/ytdl-org/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/ytdl-org/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
- [x] Checked that provided video/audio/playlist URLs (if any) are alive and playable in a browser

### What is the purpose of your *issue*?
- [x] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### Description of your *issue*, suggested solution and other information

--write-thumbnail and --list-thumbnails are only returning the 500x500 thumbnail for Soundcloud, but its possible to get the original by replacing `t500x500` in the url with `original`

Before:
https://i1.sndcdn.com/artworks-000159380373-p1pgq3-t500x500.png

After:
https://i1.sndcdn.com/artworks-000159380373-p1pgq3-original.jpg

### PR DESCRIPTION
**Pull request by Harry-Kwon**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/ytdl-org/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Fixes #20554

Added -original quality thumbnails to the soundcloud extractor

The —write-thumbnails and —list-thumbnails options will now give both -t500x500 and -original quality thumbnails

—write-thumbnail will still give -t500x500 thumbnails as it did before since the dimensions of the original quality thumbnail image is not fixed

### CODE REVIEW

File: youtube_dl/extractor/soundcloud.py [hunk 1]
```diff
@@ -217,8 +217,15 @@ def _extract_info_dict(self, info, full_title=None, quiet=False, secret_token=No
         if quiet:
             self.report_extraction(name)
         thumbnail = info.get('artwork_url') or info.get('user', {}).get('avatar_url')
+        thumbnails = []
         if isinstance(thumbnail, compat_str):
-            thumbnail = thumbnail.replace('-large', '-t500x500')
+            thumbnail = thumbnail.replace('-large', '-original')
```
Comment (dstftw) [inline youtube_dl/extractor/soundcloud.py]: All three thumbnails should be extracted.

File: youtube_dl/extractor/soundcloud.py [hunk 2]
```diff
@@ -231,6 +238,7 @@ def extract_count(key):
             'title': title,
             'description': info.get('description'),
             'thumbnail': thumbnail,
```
Comment (dstftw) [inline youtube_dl/extractor/soundcloud.py]: Remove if you use `thumbnails`.

Comment (Harry-Kwon) [reply youtube_dl/extractor/soundcloud.py]: Fixed in commit 51bfe3e

File: youtube_dl/extractor/soundcloud.py (hunk 1 above)
Comment (Harry-Kwon) [reply youtube_dl/extractor/soundcloud.py]: Fixed to extract the 3 known formats used on Soundcloud’s page, but there may be other unlisted formats available

**Pull request closed without merging**